### ISSUE TITLE
Install with laravel 8

### ISSUE THREAD

**Issue by foremtehan** [badfarm/zanzara#7]
Hey i just required it with fresh laravel 8 project and i got these : 

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - wyrihaximus/react-child-process-messenger[2.7.1, ..., 2.10.0] require doctrine/inflector ^1.0 -> found doctrine/inflector[v1.0, ..., 1.4.x-dev] but the package is fixed to 2.0.3 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

    - badfarm/zanzara 0.5.1 requires react/filesystem ^0.1.2 -> satisfiable by react/filesystem[v0.1.2, 0.1.x-dev].

    - react/filesystem[v0.1.2, ..., 0.1.x-dev] require wyrihaximus/react-child-process-pool ^1.3 -> satisfiable by wyrihaximus/react-child-process-pool[1.3.0, ..., 1.6.0].

    - wyrihaximus/react-child-process-pool 1.6.0 requires wyrihaximus/react-child-process-messenger ^2.10 -> satisfiable by wyrihaximus/react-child-process-messenger[2.10.0].

    - wyrihaximus/react-child-process-pool[1.5.0, ..., 1.5.1] require wyrihaximus/react-child-process-messenger ^2.9 -> satisfiable by wyrihaximus/react-child-process-messenger[2.9.0, ..., 2.10.0].

    - wyrihaximus/react-child-process-pool[1.3.0, ..., 1.4.2] require wyrihaximus/react-child-process-messenger ^2.7.1 -> satisfiable by wyrihaximus/react-child-process-messenger[2.7.1, ..., 2.10.0].

    - Root composer.json requires badfarm/zanzara ^0.5.1 -> satisfiable by badfarm/zanzara[0.5.1].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

```

Can you help me ?

**Comment by cheeghi** [comment]
Have you tried the same solution at #3?

**Comment by foremtehan** [comment]
It would be nice  if this package work with laravel without any extra work on composer json

**Comment by cheeghi** [comment]
Yes I agree, we added the task on the project [board](https://github.com/badfarm/zanzara/projects/1#card-50080940).

**Comment by cheeghi** [comment]
Fixed in 0.6.0
